CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# dbt_shopify_source v0.19.0
+
+[PR #108](https://github.com/fivetran/dbt_shopify_source/pull/108) includes the following updates:
+
+## Feature Update: GraphQL API Support
+
+Fivetran very recently released a new version of the Shopify connector that leverages Shopify's newer [GraphQL](https://shopify.dev/docs/apps/build/graphql) API instead of the REST API, as Shopify deprecated the REST API in October 2024. The GraphQL and REST API-based schemas are slightly different, but this package is designed to run for either or, not both. It will do so based on the value of the `shopify_api` variable.
+
+By default, `shopify_api` is set to `rest` and will run the `stg_shopify__*` models in the [rest](https://github.com/fivetran/dbt_shopify_source/tree/main/models/rest) folder. If you would like to run the package on a GraphQL-based schema, adjust `shopify_api` accordingly. This will run the `stg_shopify_gql__*` models in the [graphql](https://github.com/fivetran/dbt_shopify_source/tree/main/models/graphql) folder.
```

Comment:
Small format suggetion.
```suggestion
By default, `shopify_api` is set to `rest` and will run the `stg_shopify__*` models in the [`rest`](https://github.com/fivetran/dbt_shopify_source/tree/main/models/rest) folder. If you would like to run the package on a GraphQL-based schema, adjust `shopify_api` accordingly. This will run the `stg_shopify_gql__*` models in the [`graphql`](https://github.com/fivetran/dbt_shopify_source/tree/main/models/graphql) folder.
```